src/libs/actions/OnyxDerived/index.ts

```diff
@@ -0,0 +1,76 @@
+/**
+ * This file contains logic for derived Onyx keys. The idea behind derived keys is that if there is a common computation
+ * that we're doing in many places across the app to derive some value from multiple Onyx values, we can move that
+ * computation into this file, run it only once, and then share it across the app by storing the result of that computation in Onyx.
+ *
+ * The primary purpose is to optimize performance by reducing redundant computations. More info can be found in the README.
+ */
+import Onyx from 'react-native-onyx';
+import OnyxUtils from 'react-native-onyx/dist/OnyxUtils';
+import Log from '@libs/Log';
+import ObjectUtils from '@src/types/utils/ObjectUtils';
+import ONYX_DERIVED_VALUES from './ONYX_DERIVED_VALUES';
+
+/**
+ * Initialize all Onyx derived values, store them in Onyx, and setup listeners to update them when dependencies change.
+ */
+function init() {
+    for (const [key, {compute, dependencies}] of ObjectUtils.typedEntries(ONYX_DERIVED_VALUES)) {
+        // Create an array to hold the current values for each dependency.
+        // We cast its type to match the tuple expected by config.compute.
+        let dependencyValues = new Array(dependencies.length) as Parameters<typeof compute>[0];
+
+        OnyxUtils.get(key).then((storedDerivedValue) => {
```

Comment:
I am concerned this `OnyxUtils.get()` pattern is going to make it's way into the rest of the app and I think that is going to be a very bad pattern for people to follow. I suggest adding an ESLint rule that throws an error on these. We can disable the eslint rule where we need to, but add very explicit comments that the pattern is NOT supposed to be followed.

I've already seen we've struggled to keep the usage of `makeRequestWithSideEffects()` out of common patterns and I think the same thing is going to happen here.

Is there a reason why `get()` and `tupleGet()` are necessary? Why can we not get them from Onyx like we normally would (a connection that immediately disconnects itself)?